test/cpp/microbenchmarks/BUILD

```diff
@@ -198,47 +222,50 @@ grpc_cc_binary(
     deps = [":fullstack_unary_ping_pong_h"],
 )
 
-grpc_cc_binary(
+grpc_cc_test(
     name = "bm_metadata",
-    testonly = 1,
     srcs = ["bm_metadata.cc"],
     tags = ["no_windows"],
+    uses_polling = False,
     deps = [":helpers"],
 )
 
-grpc_cc_binary(
+grpc_cc_test(
     name = "bm_chttp2_hpack",
-    testonly = 1,
     srcs = ["bm_chttp2_hpack.cc"],
     tags = ["no_windows"],
+    uses_polling = False,
     deps = [":helpers"],
 )
 
-grpc_cc_binary(
+grpc_cc_test(
     name = "bm_opencensus_plugin",
-    testonly = 1,
     srcs = ["bm_opencensus_plugin.cc"],
     language = "C++",
     deps = [
-        ":helpers",
+        ":helpers_secure",
         "//:grpc_opencensus_plugin",
         "//src/proto/grpc/testing:echo_proto",
     ],
 )
 
-grpc_cc_binary(
+grpc_cc_test(
     name = "bm_timer",
-    testonly = 1,
     srcs = ["bm_timer.cc"],
     tags = ["no_windows"],
+    uses_polling = False,
     deps = [":helpers"],
 )
 
-grpc_cc_binary(
+grpc_cc_test(
     name = "bm_threadpool",
-    testonly = 1,
+    size = "enormous",
```

Comment:
how long does this take so you need "enormous"?

Reply:
note that foundry currently has trouble with test that take very long time: b/143515898
also, tests that take very long time are not very useful in general.

Reply:
also consider disabling the tests on mac, because our resources there are limited.

Reply:
Disabled and enormous sizes reduced.